src/appmixer/linear/core/CreateComment/CreateComment.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+module.exports = {
+    async receive(context) {
+
+        const { issueId, body } = context.messages.in.content;
+
+        // Build GraphQL mutation for creating a comment
+        const graphqlMutation = `
+            mutation CommentCreate($input: CommentCreateInput!) {
+                commentCreate(input: $input) {
+                    success
+                    comment {
+                        id
+                        body
+                        createdAt
+                        updatedAt
+                        user {
+                            id
+                            name
+                            email
+                        }
+                        issue {
+                            id
+                            title
+                        }
+                    }
+                }
+            }
+        `;
+
+        const input = {
+            issueId: issueId,
+            body: body
+        };
+
+        // https://linear.app/developers/graphql
+        const { data } = await context.httpRequest({
+            method: 'POST',
+            url: 'https://api.linear.app/graphql',
+            headers: {
+                'Authorization': `Bearer ${context.auth.accessToken}`,
+                'Content-Type': 'application/json'
+            },
+            data: {
+                query: graphqlMutation,
+                variables: { input: input }
```

Comment:
```suggestion
                variables: { input }
```

Or use issueId and body and omit `input`